src/app/main/component/user/components/profile/edit-profile/personal-photo/personal-photo.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="details-img">
-  <app-user-profile-image [additionalImgClass]="'big'" [imgPath]="avatarImg" [firstName]="userName"> </app-user-profile-image>
-  <button class="tertiary-global-button m-btn" (click)="openEditPhoto()" #editImage>
-    {{ 'user.edit-profile.edit-button' | translate }}
-  </button>
+  <app-user-profile-image [additionalImgClass]="'for-edit-page'" [imgPath]="avatarImg" [firstName]="userName"> </app-user-profile-image>
+  <a (click)="openEditPhoto()" #editImage>
+    <img src="../../../../../../../../assets/img/profile/icons/edit-photo.svg" alt="icon-edit" />
```

Comment:
You can write src shorter: `src="assets/img/profile/icons/edit-photo.svg"`

Reply:
ok, fixed it